cli/src/state/atoms/actions.ts

```diff
@@ -278,7 +291,8 @@ export const sendPrimaryButtonClickAtom = atom(null, async (get, set) => {
  */
 export const sendSecondaryButtonClickAtom = atom(null, async (get, set) => {
 	const message: WebviewMessage = {
-		type: "secondaryButtonClick",
+		type: "askResponse",
+		askResponse: "noButtonClicked",
```

Comment:
These seem surprising/scary to me, as they definitely change the structure of the message. Also we change the secondaryButton to noButton?

Reply:
This was changed in base of the type definitions that come from the extension.